etc/samples/atoms_generic.sample.yaml

```diff
@@ -0,0 +1,26 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+sds_version: gluster-3.8.3
+object_details:
+  generic_object:
+    atoms:
+        create:
+          enabled: True
+          uuid: statically generated per atom
```

Comment:
In case if they clash, what are the after effects. How we come to know that there is a clash? Some how if we know that there is a clash(parsing?) then we are good